api/v1/model/src/commonMain/kotlin/JobStatus.kt

```diff
@@ -39,5 +39,8 @@ enum class JobStatus {
     FAILED,
 
     /** The job was processed successfully. */
-    FINISHED
+    FINISHED,
+
+    /** The job has finished but there were some severe issues. */
+    FINISHED_WITH_ERRORS
```

Comment:
To align with docs and ORT terminology, better name this `FINISHED_WITH_ISSUES`? Similar for several other places / class names that use "error".